openapi/openapi.yaml

```diff
@@ -247,3 +254,202 @@ paths:
           description: The current User
         'default':
           description: Error
+components:
+  schemas:
+    Error:
+      title: 'Error'
+      description: "A JSON API Error object"
+      type: object
+      externalDocs:
+        description: 'JSON API Error Specification'
+        url: 'https://jsonapi.org/format/#errors'
+      properties:
+        errors:
+          type: array
+          title: 'ErrorItems'
+          items:
+            $ref: '#/components/schemas/ErrorItem'
+          minItems: 1
+      required:
+        - errors
+
+    ErrorItem:
+      title: 'ErrorItem'
+      description: 'An item returned in the Error "errors" value'
+      externalDocs:
+        description: 'JSON API Error Specification'
+        url: 'https://jsonapi.org/format/#errors'
+      type: object
+      properties:
+        code:
+          description: 'Unique identifier for the error'
+          type: string
+          example: 'not_found'
+        detail:
+          type: string
+          description: 'Error detail'
+          example: 'Record not found.'
+        status:
+          type: string
+          description: 'String representation of HTTP status code'
+          example: '404'
+        source:
+          type: object
+          properties:
+            parameter:
+              description: 'A string indicating which URI query parameter caused the error.'
+              type: string
+            pointer:
+              description: 'A JSON Pointer [RFC6901] to the associated entity in the request document '
+              type: string
+      required:
+        - detail
+        - status
+
+    Namespace:
+      title: 'Namespace'
+      description: 'An Ansible content Namespace'
+      type: object
+      properties:
+        name:
+          $ref: '#/components/schemas/NamespaceName'
+      required:
+       - name
+
+    NamespaceName:
```

Comment:
What's the point of having every single field extracted into its own schema?

Reply:
NamespaceName schema could be reused in the NamespaceName parameter
or anywhere namespace name validation rules apply.

This current version doesn't do that at the moment, but I can add it.